core-10/src/main/java/configuration/Specification.java

```diff
@@ -0,0 +1,15 @@
+package configuration;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.FIELD})
+
+public @interface Specification {
+
+    int lowerLimit() default 0;
```

Comment:
okay, basically looks good, but every time I have to remember what the respective value is for these digits, let's put there real specification values

also, naming is not correct, lower and upper are relative adjectives